server/controllers/article_controller.js

```diff
@@ -83,9 +83,9 @@ export const getArticleAnnotations = (user, uri, topLevelOnly) => {
       return article.annotations;
     });
   } else {
-    const deepPath = 'annotations.childAnnotations.childAnnotations.childAnnotations.childAnnotations.childAnnotations.childAnnotations';
+    const deepPath = 'annotations'.concat('.childAnnotations'.repeat(50));
     return getArticle(uri)
-    .deepPopulate(deepPath, { populate: { annotations: { match: query } } })
+    .deepPopulate(deepPath, { match: query })
```

Comment:
Yeah, but I actually changed it before I merged it to match: query where query includes parent: null, in order to incorporate the other annotation conditions above. I don't think that's the problem (unless it's not working on master either).